### PR TITLE
Correct a typo

### PR DESCRIPTION
**Pull request by gr-qft**

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| buildDuration | 9.1s | 8.8s | -359ms |
| nodeModulesSize | 82.4 MB | 82.4 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.133 | 2.141 | ⚠️ +0.01 |
| / avg req/sec | 1171.8 | 1167.94 | ⚠️ -3.86 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.162 | 1.128 | -0.03 |
| /error-in-render avg req/sec | 2151.99 | 2216.09 | +64.1 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| 677f882d2ed8..0679.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-b992ce4..2479.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-95c2..e870.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.1 kB | 59.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| polyfills-d3..23f6.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| _app-0d19cb6..5497.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-2cf0e..a0a0.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-42456f..0c06.js gzip | 887 B | 887 B | ✓ |
| index-8081ce..e44f.js gzip | 227 B | 227 B | ✓ |
| link-0ab9f83..fa00.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..c3d8.js gzip | 303 B | 303 B | ✓ |
| withRouter-0..a68e.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| index.html gzip | 615 B | 615 B | ✓ |
| link.html gzip | 620 B | 620 B | ✓ |
| withRouter.html gzip | 608 B | 608 B | ✓ |
| Overall change | 1.84 kB | 1.84 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| buildDuration | 10.3s | 10.4s | ⚠️ +64ms |
| nodeModulesSize | 82.4 MB | 82.4 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| 677f882d2ed8..0679.js gzip | 12.8 kB | 12.8 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-b992ce4..2479.js gzip | 6.56 kB | 6.56 kB | ✓ |
| webpack-95c2..e870.js gzip | 751 B | 751 B | ✓ |
| Overall change | 59.1 kB | 59.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| polyfills-d3..23f6.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| _app-0d19cb6..5497.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-2cf0e..a0a0.js gzip | 3.69 kB | 3.69 kB | ✓ |
| hooks-42456f..0c06.js gzip | 887 B | 887 B | ✓ |
| index-8081ce..e44f.js gzip | 227 B | 227 B | ✓ |
| link-0ab9f83..fa00.js gzip | 1.61 kB | 1.61 kB | ✓ |
| routerDirect..c3d8.js gzip | 303 B | 303 B | ✓ |
| withRouter-0..a68e.js gzip | 302 B | 302 B | ✓ |
| Overall change | 8.31 kB | 8.31 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 323 B | 323 B | ✓ |
| Overall change | 323 B | 323 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | gr-qft/next.js patch-3 | Change |
| - | - | - | - |
| _error.js | 1 MB | 1 MB | ✓ |
| 404.html | 2.67 kB | 2.67 kB | ✓ |
| hooks.html | 1.92 kB | 1.92 kB | ✓ |
| index.js | 1 MB | 1 MB | ✓ |
| link.js | 1.06 MB | 1.06 MB | ✓ |
| routerDirect.js | 1.05 MB | 1.05 MB | ✓ |
| withRouter.js | 1.05 MB | 1.05 MB | ✓ |
| Overall change | 5.16 MB | 5.16 MB | ✓ |

</details>

</details>
Commit: 47acb949a706f6008c19e8c2bea15e01f2e0101f

File: docs/api-reference/next.config.js/environment-variables.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ For example, the following line:
 return <h1>The value of customKey is: {process.env.customKey}</h1>
 ```
 
-Will end up being:
+will end up being:
```
Comment (lfades) [inline docs/api-reference/next.config.js/environment-variables.md:44]: I'm not sure if this is right. In our curernt style every paragraph always starts with a uppercase.

Comment (gr-qft) [reply docs/api-reference/next.config.js/environment-variables.md:44]: I don't think it's correct to begins every paragraph with an uppercase. The starting paragraph, the code that follows it, and the paragraph that follows the code make a single unit.

Comment (faith) [inline docs/api-reference/next.config.js/environment-variables.md:44]: Should be uppercase probably as any paragraph start tho lower case is easier to read i guess

**Review by 4ldas — CHANGES_REQUESTED** [review]

**Comment by timneutkens** [comment]
I'm going to close this PR as the current layout is fine as-is.

**Pull request closed without merging**